### PR TITLE
Rajkumar562: Task 1, 2, 3, 4, 5 and 6

### PR DESCRIPTION
**Pull request by Rajkumar562**
Leave any suggestions to make any changes.

### CODE REVIEW

File: Rajkumar562/style.css
Comment (PradyumnaKrishna) [inline Rajkumar562/style.css:1]: We won't need this, Use style in document.

**Review by PradyumnaKrishna — CHANGES_REQUESTED** [review]

File: Rajkumar562/style.css
Comment (Rajkumar562) [reply Rajkumar562/style.css:1]: Updated the same as requested. Please look into it.

File: Rajkumar562/task2.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  box-sizing: border-box;
+  bottom:50px;
+right: 50px;
+}
+</style>
+</head>
+<body>
+  <div></div>
+  <img class="retro" src="https://github.com/CC-BHU/web-development/raw/main/assets/images/task2.jpg" alt="retro city img">
+<div class="text">Welcome to the Retro City, looks mesmerising isn't it?</div>
```
Comment (PradyumnaKrishna) [inline Rajkumar562/task2.html:36]: This text-box isn't relative to the image, try adding all this to a parent div that have `position: absolute;`.

File: Rajkumar562/task4.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task 4</title>
+    <style>
```
Comment (PradyumnaKrishna) [inline Rajkumar562/task4.html:8]: I have provided a CSS to style this task.

**Review by PradyumnaKrishna — CHANGES_REQUESTED** [review]
Please make the following changes.

**Comment by Rajkumar562** [comment]
Made the requisite changes.

**Review by PradyumnaKrishna — APPROVED** [review]
Amazing Work @Rajkumar562!

**Comment by PradyumnaKrishna** [comment]
After syncing you fork to CC-BHU/main, discard all the commits.
Use command
```bash
git reset --hard origin/main
```
It will fix your local git repo.

**Pull request merged**